pydm/data_plugins/epics_plugins/pyepics_plugin_component.py

```diff
@@ -1,10 +1,10 @@
 import epics
 import logging
 import numpy as np
+from pydm import data_plugins
```

Comment:
For some reason this fails for me on Python 2.7.13.  Strangely, if I do `import pydm.data_plugins` it works.